README.md

```diff
@@ -18,7 +18,7 @@ _Collaborate and work together on GitHub._
   Do not use quotes on the <details> tag attributes.
 -->
 
-<!--step0-->
+<!--step0
```

Comment:
This blue plus sign, when you mouse over a changed line, let's you add an in-line comment